src/main/resources/com/google/api/codegen/py/method_sample.snip

```diff
@@ -1,27 +1,38 @@
 @extends "py/common.snip"
 
-# Generate code samples for method-level documentation.
-@snippet sampleCode(apiMethod)
+# Generate brief code examples for method-level documentation.
+@snippet exampleCode(apiMethod, initCode)
   Example:
-      @join line : util.getDocLines(sampleCode_(apiMethod))
+      @join line : util.getDocLines(sampleCode_(apiMethod, initCode))
           {@util.exampleLine(line)}
       @end
 @end
 
+# Generate code samples for standalone samples.
+@snippet sampleCode(apiMethod, initCode)
+      @join line : util.getDocLines(sampleCode_(apiMethod, initCode))
+          {@line}
+      @end
+@end
+
 @snippet sampleCode_(apiMethod)
-  {@importList(apiMethod.initCode.importSection.appImports)}
+  {@sampleCode_(apiMethod, apiMethod.initCode)}
+@end
+
+@snippet sampleCode_(apiMethod, initCode)
```

Comment:
Use a more informative naming scheme than adding `_`.

Reply:
`samplecode_` is the name that was previously being used. I am just providing an override for this method.

I fully support a better name, but since this was pre-existing, maybe we should do that in a separate PR.

Reply:
Oh! I didn't notice that. Carry on...